quattor/filesystems.pan

```diff
@@ -38,7 +38,9 @@ type structure_filesystem = {
     "mountpoint": string
     "mount"     : boolean
     "mountopts" : string = "defaults" # "Mount options"
-    "type"      : string with match (SELF, "^(ext[2-4]|reiserfs|reiser4|xfs|swap|vfat|jfs|ntfs|tmpfs|vxfs|zfs|none)$") # "Filesystem type."
+    "type"      : string with match (SELF, "^(auto|ext[2-4]|reiserfs|" +
+                    "reiser4|xfs|swap|vfat|jfs|ntfs|tmpfs|vxfs|zfs|none)$")
+                    # "Filesystem type"
```

Comment:
@ned21 I think the indent checker would be happy with the following, but I don't really mind.

```pan
    "type"      : string with match (SELF,
        "^(auto|ext[2-4]|reiserfs|reiser4|xfs|swap|vfat|jfs|ntfs|tmpfs|vxfs|zfs|none)$"
    ) # "Filesystem type"
```

Reply:
or use `choice` and split it like a long list?